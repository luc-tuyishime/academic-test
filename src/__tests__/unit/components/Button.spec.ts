import { mount } from "@vue/test-utils"
import DarkModeButton from "../../../components/DarkMode.vue"

describe("DarkModeButton", () => {
  it("Test button properly ", () => {
    const wrapper = mount(DarkModeButton)

    expect(wrapper.element).toMatchSnapshot()
  })
})
