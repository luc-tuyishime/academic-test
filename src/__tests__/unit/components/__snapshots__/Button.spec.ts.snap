// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkModeButton Test button properly  1`] = `
<button
  class="px-4 py-2 text-white bg-blue-900 rounded dark:bg-red-400"
>
  Dark Mode
</button>
`;
